### PR TITLE
test: Adds tests and storybook to CopyToClipboard component

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
- Adds tests and storybook to `CopyToClipboard` component
- Adds Redux `Provider` support for stories
- Changes `height` type to `string` in `AsyncAceEditor` story
- Changes parameter `disabled` to `disable` in stories

### TEST PLAN
1 - Execute `CopyToClipboard` tests
2 - All tests should pass

@rusackas @junlincc 

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/components/CopyToClipboard/index.jsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ class CopyToClipboard extends React.Component {
 
   renderLink() {
     return (
-      <span>
+      <span style={{ display: 'inline-flex', alignItems: 'center' }}>
```
Comment (rusackas) [inline superset-frontend/src/components/CopyToClipboard/index.jsx:105]: I'm not a huge fan of inline styles if there's a better place to stick this.

Comment (rusackas) [reply superset-frontend/src/components/CopyToClipboard/index.jsx:105]: ... at the very least, we could set up the `css={theme => css...` boilerplate to make it look like real CSS with theme support.

Comment (michael-s-molina) [reply superset-frontend/src/components/CopyToClipboard/index.jsx]: Changed to use the `css` property instead of `style`.

**Review by rusackas — APPROVED** [review]
LGTM! Thanks!

**Comment by junlincc** [comment]
Thanks for the PR! 
@pkdotson Phillip, please take a second look, then ready to go:)

**Pull request merged**